src/ontology/slurp/omim.tsv

```diff
@@ -1,56 +1,387 @@
+MONDO:0859063	removed from database	OMIM:212600	MONDO:equivalentTo	removed from database		
+MONDO:0859064	removed from database	OMIM:212700	MONDO:equivalentTo	removed from database		
+MONDO:0859065	removed from database	OMIM:221000	MONDO:equivalentTo	removed from database		
+MONDO:0859066	removed from database	OMIM:225360	MONDO:equivalentTo	removed from database		
+MONDO:0859067	removed from database	OMIM:231400	MONDO:equivalentTo	removed from database		
+MONDO:0859068	removed from database	OMIM:231610	MONDO:equivalentTo	removed from database		
+MONDO:0859069	removed from database	OMIM:239400	MONDO:equivalentTo	removed from database		
+MONDO:0859070	removed from database	OMIM:240100	MONDO:equivalentTo	removed from database		
+MONDO:0859071	removed from database	OMIM:242800	MONDO:equivalentTo	removed from database		
+MONDO:0859072	removed from database	OMIM:245500	MONDO:equivalentTo	removed from database		
+MONDO:0859073	removed from database	OMIM:249800	MONDO:equivalentTo	removed from database		
+MONDO:0859074	removed from database	OMIM:252030	MONDO:equivalentTo	removed from database		
+MONDO:0859075	removed from database	OMIM:253500	MONDO:equivalentTo	removed from database		
+MONDO:0859076	removed from database	OMIM:261660	MONDO:equivalentTo	removed from database		
+MONDO:0859077	removed from database	OMIM:264810	MONDO:equivalentTo	removed from database		
+MONDO:0859078	removed from database	OMIM:275600	MONDO:equivalentTo	removed from database		
+MONDO:0859079	removed from database	OMIM:300900	MONDO:equivalentTo	removed from database		
+MONDO:0859080	intellectual developmental disorder, x-linked, syndromic, with pigmentary mosaicism and coarse facies	OMIM:301066	MONDO:equivalentTo	intellectual developmental disorder, x-linked, syndromic, with pigmentary mosaicism and coarse facies		
+MONDO:0859081	chromosome xq13 duplication syndrome	OMIM:301069	MONDO:equivalentTo	chromosome xq13 duplication syndrome		
+MONDO:0859082	thrombophilia, x-linked, due to factor 8 defect	OMIM:301071	MONDO:equivalentTo	thrombophilia, x-linked, due to factor 8 defect		
+MONDO:0859083	systemic lupus erythematosus 17	OMIM:301080	MONDO:equivalentTo	systemic lupus erythematosus 17		
+MONDO:0859084	epilepsy, x-linked 2, with or without impaired intellectual development and dysmorphic features	OMIM:301091	MONDO:equivalentTo	epilepsy, X-linked 2, with or without impaired intellectual development and dysmorphic features		
+MONDO:0859085	neurodevelopmental disorder with gait disturbance, dysmorphic facies, and behavioral abnormalities, x-linked	OMIM:301094	MONDO:equivalentTo	neurodevelopmental disorder with gait disturbance, dysmorphic facies, and behavioral abnormalities, X-linked		
+MONDO:0859086	intellectual developmental disorder, x-linked 110	OMIM:301095	MONDO:equivalentTo	intellectual developmental disorder, X-linked 110		
+MONDO:0859087	removed from database	OMIM:301880	MONDO:equivalentTo	removed from database		
```

Comment:
What are all these "removed from database" rows?

Reply:
I was surprised to see that too. This is a message I see for OMIM deprecated terms, so I'm not sure why they are here. I'll look and double check if these are indeed deprecated and, if so, why they are being included here (they shouldn't be).